apps/client/ui/Header/Header.tsx

```diff
@@ -15,26 +15,30 @@ const mockedUserData = {
 export const Header: FC = () => {
   const [activeMenuItem, setActiveMenuItem] = useState<string>(null)
   
-  const handleMenuClick = (item: string) => {
+  const handleMenuHover = (item: string) => {
     setActiveMenuItem((prevItem) => {
       return prevItem !== item ? item : null
     })
   }
 
-  const handleDropdownMouseLeave = () => {
+  const resetActiveMenuItem = () => {
     setActiveMenuItem(null)
   }
+  
+  const handleDropdownMouseLeave = resetActiveMenuItem
```

Comment:
no need for these aliases 👌 passing just `resetActiveMenuItem` to necessary props is completely fine 😁